etcdserver/apply.go

```diff
@@ -743,8 +743,8 @@ func (a *applierV3backend) AuthDisable() (*pb.AuthDisableResponse, error) {
 }
 
 func (a *applierV3backend) AuthStatus() (*pb.AuthStatusResponse, error) {
-	a.s.AuthStore().AuthStatus()
-	return &pb.AuthStatusResponse{Header: newHeader(a.s)}, nil
```

Comment:
etcd should always return the header field. Could you update for returning both of the header and status?